src/ejabberd_ctl.erl

```diff
@@ -151,35 +145,25 @@ process(["help" | Mode]) ->
     {MaxC, ShCode} = get_shell_info(),
     case Mode of
         [] ->
-            print_usage_old(dual, MaxC, ShCode),
+            print_usage(dual, MaxC, ShCode),
```

Comment:
I would remove the entire `process(["help" | Mode])` clause, because it was supposed to provide help for the deprecated commands, while there are no such commands anymore. The `--dual` and `--long` option were undocumented anyway, and calling `mongooseimctl` without arguments is enough to get help.